mysql-test/suite/innodb/t/restart.test

```diff
@@ -77,3 +77,28 @@ SELECT * FROM tr;
 SELECT * FROM tc;
 SELECT * FROM td;
 DROP TABLE tr,tc,td;
+
+--echo #
+--echo # MDEV-27467 innodb to enfore the minimum innodb_buffer_pool_size in SET (resize) the same as startup
+--echo #
+
+let $wait_timeout = 180;
+let $wait_condition =
+  SELECT SUBSTR(variable_value, 1, 34) = 'Completed resizing buffer pool at '
+  FROM information_schema.global_status
+  WHERE LOWER(variable_name) = 'innodb_buffer_pool_resize_status';
+
+SELECT @@innodb_buffer_pool_size INTO @innodb_buffer_pool_size_orig;
+SELECT CEILING((256 + 64) * @@innodb_page_size / 1048576) * 1048576 INTO @min_pool_size;
+--replace_regex /value of '\d+/value of 'WRONG_VALUE/
+--error ER_WRONG_VALUE_FOR_VAR
+EXECUTE IMMEDIATE 'SET GLOBAL innodb_buffer_pool_size = ?' USING (@min_pool_size -1);
+
+--replace_regex /least \d+/least MIN_VAL/ /innodb_page_size=\d+/innodb_page_size=PAGE_SIZE/ /value of '\d+/value of 'WRONG_VALUE/
+SHOW WARNINGS;
```

Comment:
Instead of filtering out the values, I would prefer to add `.rdiff` files for the different values so that any inadvertent change to the limits in the future will be caught by this test. The basic `.result` file should be recorded with the default page size (`./mtr --record innodb.restart,16k`).